src/Security/Authentication/Authenticator.php

```diff
@@ -94,4 +93,14 @@ protected function hasRequiredFields(array $array): bool
 
         return true;
     }
+
+    protected function getRequest(Credentials $credentials): Request
+    {
+        return new Request(
```

Comment:
but hmmm... we can create one and then check if it's doable at all

I'll create it